src/app/fyle/split-expense/split-expense.page.ts

```
@@ -999,6 +1011,91 @@
+      services: {
+        launchDarklyService: this.launchDarklyService,
+        projectsService: this.projectsService,
+      },
+    }).pipe(takeUntil(this.destroy$));
+
+    this.categories$.pipe(takeUntil(this.destroy$)).subscribe((categories) => {
+      const categoryControl = splitForm.get('category');
+      const currentCategory = categoryControl.value as OrgCategory;
+      if (
+        currentCategory &&
+        currentCategory.id &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)